benchmarks/execute/src/main.rs

```diff
@@ -173,30 +182,50 @@ fn main() -> Result<()> {
 
             let exe = VmExe::from_elf(elf, transpiler)?;
 
+            let vm = VirtualMachine::new(default_engine(), vm_config.clone());
+            let pk = vm.keygen();
             let (widths, interactions): (Vec<usize>, Vec<usize>) = {
-                let vm = VirtualMachine::new(default_engine(), vm_config.clone());
-                let pk = vm.keygen();
                 let vk = pk.get_vk();
                 vk.inner
                     .per_air
                     .iter()
                     .map(|vk| {
-                        // TODO(ayush): figure out which width to use
-                        // let total_width = vk.params.width.preprocessed.unwrap_or(0)
-                        //     + vk.params.width.cached_mains.iter().sum::<usize>()
-                        //     + vk.params.width.common_main
-                        //     + vk.params.width.after_challenge.iter().sum::<usize>();
-                        let total_width = vk.params.width.main_widths().iter().sum::<usize>();
+                        let total_width = vk.params.width.preprocessed.unwrap_or(0)
+                            + vk.params.width.cached_mains.iter().sum::<usize>()
+                            + vk.params.width.common_main
+                            // TODO(ayush): no magic value 4. should come from stark config
+                            + vk.params.width.after_challenge.iter().sum::<usize>() * 4;
                         (total_width, vk.symbolic_constraints.interactions.len())
                     })
                     .unzip()
             };
 
             let executor = VmExecutor::new(vm_config);
-            executor
-                .execute_e1(exe.clone(), vec![])
-                // .execute_metered(exe.clone(), vec![], widths, interactions)
-                .expect("Failed to execute program");
+
+            // E2 to find segment points
```

Comment:
this was just for testing? don't we want to keep the benchmark E1?

Reply:
Yes, I'll fix this. Though with codspeed, we can probably remove the old execute tests that were using this file